api/test/api/performance_manager_test.exs

```diff
@@ -1,23 +1,70 @@
 defmodule Api.PerformanceManagerTest do
   use Api.DataCase
 
-  alias Api.PerformanceManager
+  describe "metadata log table" do
+    alias Api.Repo
+    alias Api.PerformanceManager.MetadataLog
 
-  describe "metadata_log" do
-    import Api.PerformanceManagerFixtures
+    test "metadata log schema formatted correctly" do
+      Repo.all(MetadataLog)
+    end
+  end
```

Comment:
for testing the metadata log i'm just checking that we can run `Repo.all` on it. if there is a discrepancy between our described schema and the table in the db this will trigger an error. since the db is empty, we can't test against any of the values like we were doing before.